### PR TITLE
add Number of Ways to Select Buildings

### PR DESCRIPTION
**Pull request by alexpantyukhin**
#### Description of Change

Add a guide for new LeetCode solutions.

#### Checklist

- [x] Added description of change
- [x] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#File-Name-guidelines)
- [ ] Added tests and example, test must pass
- [x] Relevant documentation/comments is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.

Notes:

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🚀

File: leetcode/src/2222.c [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        }
+
+        secondItemsNumber += firstItemsNumber;
+    }
+    
+    return result;
+        
+}
+
+// Counting: Calculating sum of first ways of building. Then plus it to second and calculate with third.
```
Comment (tjgurwara99) [inline leetcode/src/2222.c:22]:
```suggestion
// numberOfWays returns the sum of number ways of selecting first building
// and the number of ways of selecting second building which gives us the
// number of ways of selecting three building such that no
// consecutive buildings are in the same category.
```

File: leetcode/src/2222.c [hunk 2]
```diff
@@ -0,0 +1,27 @@
+long long numberOfWaysForChar(char * s, char c){
```
Comment (tjgurwara99) [inline leetcode/src/2222.c:1]: Non blocking comment, sometimes `leetcode` names things that make sense in the context of the problem but not generally in the context of code (just my opinion 😂) therefore when working on code that would be read by more people, it is usually better to name things appropriately to help people reading your code.

Things like `firstItemsNumber` and `secondItemsNumber` doesn't explain anything when reading this code and have to be "deciphered" by the reader. So naming things appropriately is very much appreciated but again this is not a blocking comment.

**Review by tjgurwara99 — CHANGES_REQUESTED** [review]

**Review by tjgurwara99 — DISMISSED** [review]

File: leetcode/src/2222.c (hunk 2 above)
Comment (alexpantyukhin) [reply leetcode/src/2222.c]: fixed, please review.

**Review by tjgurwara99 — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**